SCloudRandomizer/MainViewController.m

```diff
@@ -354,7 +369,7 @@ - (void)getTrackInfo:(NSDictionary *)track shouldPlay:(BOOL)play
 
 }
 
-// TODO: Load songs based on SearchParams
+// Load songs based on SearchParams
 // Refresh/reload track list
 - (void)refreshTrackList
```

Comment:
Same comment as before, we can update this to take a url in as well. Lets discuss it!!